### ISSUE TITLE
Error: Base-level rules cannot contain the parent-selector-referencing character '&'.

### ISSUE THREAD

**Issue by Imaginaerum** [zurb/foundation-sites#7669]
I want to use foundation.

I use foundation-sites 6.1

Error:

    Running "sass:manager" (sass) task
    Error: Base-level rules cannot contain the parent-selector-referencing character '&'.
            on line 118 of /path/to/bower_components/foundation-sites/scss/grid/_classes.scss, in `@content'
            from line 220 of /path/to/bower_components/foundation-sites/scss/util/_mixins.scss, in `@content'
            from line 133 of /path/to/bower_components/foundation-sites/scss/util/_breakpoint.scss, in `breakpoint'
            from line 219 of /path/to/bower_components/foundation-sites/scss/util/_mixins.scss, in `-zf-each-breakpoint'
            from line 84 of /path/to/bower_components/foundation-sites/scss/grid/_classes.scss, in `foundation-grid'
            from line 53 of /path/to/bower_components/foundation-sites/scss/foundation.scss, in `foundation-everything'
            from line 3 of /path/to/bower_components/foundation-sites/assets/foundation.scss
            from line 7 of /path/to/scss/css/mycss.scss
      Use --trace for backtrace.
    Warning: Exited with error code 65 Use --force to continue.

My gruntfile config:

    ...
    sass: {
        manager: {
            options: {
                compass: true,
                style: 'expanded',
                loadPath: [
                    '/path/to/bower_components/foundation-sites',
                ]
            },
            files: {
                '/path/to/css/mycss.css': '/path/to/scss/css/mycss.scss'
            }
        }
    },
    ...
    
Here my gem list

    *** LOCAL GEMS ***

    celluloid (0.16.0)
    chunky_png (1.3.5)
    compass (1.1.0.alpha.3, 1.0.3)
    compass-core (1.1.0.alpha.3, 1.0.3)
    compass-import-once (1.0.5)
    ffi (1.9.10)
    hitimes (1.2.3)
    listen (2.10.1)
    multi_json (1.11.2)
    rake (10.4.2)
    rb-fsevent (0.9.6)
    rb-inotify (0.9.5)
    sass (3.4.20, 3.2.0)
    timers (4.0.4)
    
I try to use sass 3.2 but when I install compass, sass version is 3.4.20...

**Comment by jfalk** [comment]
I'm encountering this same issue with flex grid.

**Comment by rjulio** [comment]
Im getting the same error using 6.1.0

My gruntfile.js
```
sass:{
            dist:{
                options:{
                    style:'compressed',
                    loadPath: ['<%= dirs.dev %>/scss/vendor/foundation'],
                    update:true
                },
                files:[{
                    expand:true,
                    cwd:'<%= dirs.dev%>/scss/',
                    src:['*.scss'],
                    dest:'<%= dirs.prod %>/css',
                    ext:'.css'
                }]
            }
        },
```

my package.json
```
"dependencies": {
    "font-awesome": "^4.5.0",
    "foundation-sites": "^6.1.0",
    "slider-pro": "^1.2.3"
  },
  "devDependencies": {
    "grunt": "^0.4.5",
    "grunt-contrib-copy": "^0.8.2",
    "grunt-contrib-sass": "^0.9.2",
    "grunt-contrib-uglify": "^0.11.0",
    "grunt-contrib-watch": "^0.6.1"
  }
```

**Comment by gakimball** [comment]
This was fixed with d8a8bbec32695a81b86134f8e40647237b4e96a1 and then re-introduced accidentally with 0bc98a883f01695c9a13177057e38379108d8531. Fixed it again!

This is the team's last day in the office, so we'll put out 6.1.1 today with this fix.

**Comment by Imaginaerum** [comment]
Thanks a lot :)
